src/serializeBinary.ts

```diff
@@ -37,9 +37,17 @@ const ArrayBufferSerializer: Serializer<ArrayBuffer, string> = {
   // from https://developers.google.com/web/updates/2012/06/How-to-convert-ArrayBuffer-to-and-from-String
   // modified to use Int8Array so that we can hold odd number of bytes
   toObject: async (ab: ArrayBuffer) => {
-    return new Int8Array(ab).reduce(function (data, byte) {
-      return data + String.fromCharCode(byte);
-    }, "");
+    const bytes = new Int8Array(ab);
+    const MAX_BUFFER_LENGTH = 8 * 1024;
```

Comment:
why 8k? whats the buffer sizes you have issues with?
also please add code comment about the bug you encountered and why this buffer splitting strategy is required to solve it

Reply:
Sorry, 64k is the buffer sizes I have issue with. I just re-commit again. This is for large base64 string.